web/less/base.less

```diff
@@ -1,8 +1,8 @@
-@import 'flexbox.less';
+@import "flexbox.less";
```

Comment:
@Raulkg Are you using some automatic linter that is making all of these style changes? It makes it a bit harder to find the changes that are relevant to the issues being fixed.